Cloud Application Manager/Deploying Anywhere/using-microsoft-azure.md

```diff
@@ -42,8 +39,118 @@ You need an Microsoft Azure subscription to be able to consume Azure services. F
+| Instances | Specify the number of instances to spawn. Note that at this time, we don’t autoscale or load balance instances. To enable that, you have to manually configure these options in [Azure](https://msdn.microsoft.com/en-us/library/hh680914). |
+
+
+**Network**
+
+| Option | Description |
+|--------|-------------|
+| Virtual Network |	This network has to be created before because Cloud Application Manager doesn't deploy any. |
+| Subnet | This subnet is the resource related to  the virtual machine's network interface. Actually a virtual network is not used at deployment time. |
+
+If you can't create any policy box on Windows Azure provider probably you have to create a virtual network from azure portal or you may deploy a new one with a template as we describe in following section.
+
+**Microsoft Azure ARM Template Deployment Options**
+
+Azure ARM Templates are supported on Cloud Application Manager with Microsoft Azure provider. You can deploy whatever you want with the same syntax you use on Azure APIs and portal. For this purpose you can create a custom deployment policy and deploy it with an ARM Template box together.
+
+**Steps** 
+
+1. Create Deployment Policy:
+   * Go to **Boxes** > **New** > **Deployment Policy**.
+   ![microsoft-azure-create-new-deployment-policy-5.png](../../images/cloud-application-manager/microsoft-azure-create-new-deployment-policy-5.png)
+   * **Select Azure Resource Manager** on the menu.
+   ![microsoft-azure-select-arm-new-deployment-policy-box-6.png](../../images/cloud-application-manager/microsoft-azure-select-arm-new-deployment-policy-box-6.png)
+   * Select provider, name and description fields.
+   ![microsoft-azure-select-provider-name-7.png](../../images/cloud-application-manager/microsoft-azure-select-provider-name-7.png)
+   * Click **Save**.
+
+2. Edit Deployment Policy.
+   * Go to **Template** > **Code** and press **Edit**.
+   * Put your own template and click **Save**.
+
+3. Deploy Template.
+   ![microsoft-azure-deploy-template-8.png](../../images/cloud-application-manager/microsoft-azure-deploy-template-8.png)
+   * From Template click **Deploy**.
+   * **Select** one ARM Template Box.
+   * Choose a name and description for the new instance.
+   * Press **Deploy**.
+
+### Registering Existing Instances from your Azure Account
+
+You can import existing Virtual Machines into you workspace only in one click. The list of available instances you can import come with your Microsoft Azure provider synchronization.
+
+**Available Instances**
+
+As part of the result of synchronization process you can find a list of available virtual machines that already exist in your account but not used yet in Cloud Application Manager. You can import an existing one clicking **Import** button.
+
+The only requirement is instance must be in Running status.
+
+![microsoft-azure-available-instances-9.png](../../images/cloud-application-manager/microsoft-azure-available-instances-9.png)
 
+<<<<<<< HEAD
```

Comment:
check this as i'm unclear what you are trying to place here in markdown.  this isn't rendering.